src/gsy_e/gsy_e_core/sim_results/file_export_endpoints.py

```diff
@@ -171,17 +172,26 @@ def labels(self) -> List:
         ] + self._specific_labels()
 
     def _specific_labels(self):
-        if isinstance(self.area.strategy, StorageStrategy):
+        if type(self.area.strategy) is StorageStrategy:
             return [
                 "bought [kWh]",
                 "sold [kWh]",
                 "charge [kWh]",
                 "offered [kWh]",
                 "charge [%], losses[kWh]",
             ]
-        if isinstance(self.area.strategy, LoadHoursStrategy):
```

Comment:
This actually changes the behaviour of this code, please have a look here:
https://switowski.com/blog/type-vs-isinstance/#difference-between-isinstance-and-type
We also want to export these labels and rows for all parent classes of LoadHoursStrategy (e.g. DefinedLoadStrategy). This is why we need isinstance here. This also applies to all other if blocks here. Please revert. Thanks!